journals/apps/journals/templates/journals/journal_page.html

```diff
@@ -1,17 +1,45 @@
 {% extends "base.html" %}
+{% load static %}
 
 {% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/journal_page.css' %}">
+{% endblock %}
+
+
 
 {% block content %}
+    <script>
+        let journal_structure = {{page.specific.get_json_journal_structure |safe}};
+    </script>
     <div id="journal-sub-page" class="container content">
+
         <div class="row">
-            <div class="col-12">
-                <h1>{{ page.title }}</h1>
-                    {% for child in page.get_children %}
-                    <a href="{{child.url}}">{{child.title}}</a>
-                    {% endfor %}
-                    <div>{{page.body}}</div>
+            <div class="col-0 col-lg-3 d-none d-lg-block">
+                <table class="page-side-menu">
+                    <tbody>
+                        <tr>
+                            <td class="toc-journal-name">{{page.get_parent_journal}}</td>
+                        </tr>
+                        <tr>
+                            <td class="toc-journal-struct"></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+
+
+            <div class="col-12 col-lg-9 col-xl-6">
+                <h1 class="page-title">{{page.title}}</h1>
+                <div>{{page.body}}</div>
             </div>
         </div>
     </div>
 {% endblock %}
+
+{% block extra_js %}
+    <script type="text/javascript" src="{% static 'js/lib/youtube.min.js' %}"></script>
```

Comment:
youtube js not needed?

Reply:
Whoops, that was from the other work. Forgot to pull it out.